code/_helpers/maths.dm

```diff
@@ -138,3 +138,17 @@
 
 /proc/rand_between(var/lower, var/upper)
 	return (rand() * (upper - lower)) + lower
+
+
+/proc/dist3D(var/atom/A, var/atom/B)
```

Comment:
This finds distances accurately across Zlevels. At some point soon i'll get around to writing 3D range procs too